__tests__/lint/assertions-ref-pattern-error/snapshot.js

```diff
@@ -0,0 +1,116 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E lint assertions-ref-pattern-error 1`] = `
+
+validating /openapi.yaml...
+[1] openapi.yaml:12:3 at #/paths/~1pets
+
+PathItem should contain ref to components folder
+
+10 |   - url: http://petstore.swagger.io/v1
+11 | paths:
+12 |   /pets:
+   |   ^^^^^
+13 |     get:
+14 |       summary: List all pets
+
+Error was generated by the ref-pattern-no-properties assertion rule.
+
+
+[2] openapi.yaml:45:17 at #/paths/~1pets/get/responses/default/content/application~1json/schema
+
+Response MediaType schema should contain ref to components folder
+
+43 |           application/json:
+44 |             schema:
+45 |               $ref: ./components/schemas/Error.yaml
+   |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This looks like schema does contain ref to the components folder. Why error here?